src/metrics.ts

```diff
@@ -328,10 +315,9 @@ export function getMetrics (
       labelNames: ['hit']
     }),
 
-    asyncValidationDelayFromFirstSeenSec: register.histogram<{ topic: TopicLabel }>({
+    asyncValidationDelayFromFirstSeenSec: register.histogram({
```

Comment:
Label value is not set for this metric
https://github.com/ChainSafe/js-libp2p-gossipsub/blob/eb53ea96a765a4aeed19b4feee273126a8286616/src/metrics.ts#L793

Topic might not be available when setting the metric hence I updated the type instead of updating the code and setting the topic.